multigeiger/switches.cpp

```diff
@@ -6,19 +6,18 @@
 #include "switches.h"
 
 // Inputs for the switches
-#if CPU == STICK
-#define PIN_SWI_0 36
-#define PIN_SWI_1 37
-#define PIN_SWI_2 38
-#define PIN_SWI_3 39
-#else
-#define PIN_SWI_0 39
-#define PIN_SWI_1 38
-#define PIN_SWI_2 37
-#define PIN_SWI_3 36
-#endif
-
-void setup_switches() {
+static unsigned int PIN_SWI_0 = 39;
+static unsigned int PIN_SWI_1 = 38;
+static unsigned int PIN_SWI_2 = 37;
+static unsigned int PIN_SWI_3 = 36;
+
+void setup_switches(bool lora) {
```

Comment:
similar here, it is rather the board hardware (aka "CPU") that influences this than lora.